addons/event/data/mail_template_data.xml

```diff
@@ -10,18 +10,224 @@
+    <tr>
+        <td align="center" style="min-width: 590px;">
+            <table width="590" border="0" cellpadding="0" cellspacing="0" style="min-width: 590px; background-color: white; padding: 0px 8px 0px 8px; border-collapse:separate;">
+                <tr><td valign="top" style="font-size: 14px;">
+                    <table style="width:100%;">
+                        <tr>
+                            <td style="vertical-align:top;">
+                                <img src="/web_editor/font_to_img/61555/rgb(81,81,102)/34" style="padding:4px;max-width:inherit;" height="34" alt=""/>
+                            </td>
+                            <td style="padding: 0px 10px 0px 10px;width:50%;line-height:20px;vertical-align:top;">
+                                <div><strong>From</strong> <t t-out="object.event_id.date_begin_located or ''">May 4, 2021, 7:00:00 AM</t></div>
+                                <div><strong>To</strong> <t t-out="object.event_id.date_end_located or ''">May 6, 2021, 5:00:00 PM</t></div>
+                                <div style="font-size:12px;color:#9e9e9e"><i>(<t t-out="object.event_id.date_tz or ''">Europe/Brussels</t>)</i></div>
+                            </td>
+                            <td style="vertical-align:top;">
+                                <t t-if="event_address">
+                                    <img src="/web_editor/font_to_img/61505/rgb(81,81,102)/34" style="padding:4px;max-width:inherit;" height="34" alt=""/>
+                                </t>
+                            </td>
+                            <td style="padding: 0px 10px 0px 10px;width:50%;vertical-align:top;">
+                                <t t-if="event_address">
+                                    <t t-set="location" t-value="''"/>
+                                    <t t-if="object.event_id.address_id.name">
+                                        <div t-out="object.event_id.address_id.name or ''">Teksa SpA</div>
+                                    </t>
+                                    <t t-if="object.event_id.address_id.street">
+                                        <div t-out="object.event_id.address_id.street or ''">Puerto Madero 9710</div>
+                                        <t t-set="location" t-value="object.event_id.address_id.street"/>
+                                    </t>
+                                    <t t-if="object.event_id.address_id.street2">
+                                        <div t-out="object.event_id.address_id.street2 or ''">Of A15, Santiago (RM)</div>
+                                        <t t-set="location" t-value="'%s, %s' % (location, object.event_id.address_id.street2)"/>
+                                    </t>
+                                    <div>
+                                    <t t-if="object.event_id.address_id.city">
+                                        <t t-out="object.event_id.address_id.city or ''">Pudahuel</t>,
```

Comment:
no need `or ''` because you check above that's not falsy (check other place in the templates)